image_inspection.tex

```diff
@@ -1,2 +1,88 @@
 \section{Image Inspection}
-\label{sec:image_inspection}
\ No newline at end of file
+\label{sec:image_inspection}
+
+The prospects for human-based inspection of the vast number of images to be
+produced by the Rubin/LSST are unavoidably going to be limited to a very small
+fraction of the dataset produced (even nightly, let alone for the full 10-year
+survey).  Yet, the potential value of getting human eyes on the images (including
```

Comment:
I think this depends a bit on what you mean by "very small fraction". I think that human inspection of every exposure at the focal plane level is not too intractable (1000 per night at 2s per exposure?).

Reply:
Thanks.  I removed the “very small” qualifier to “fraction”.